library/vessels/floating_barge.yaml

```diff
@@ -1,14 +1,7 @@
 crane_specs:
   max_lift: 500             # t
-  radius: 15                # m
-jacksys_specs:              # TEMPORARY UNTIL FLOATING FUNCTIONALY ADDED
-  air_gap: 10               # m, distance from MSL to jacked up height
-  leg_length: 85            # m
-  leg_pen: 5                # m
-  max_depth: 1000            # m
-  max_extension: 1000        # m
-  speed_above_depth: 10000    # m/min
-  speed_below_depth: 10000   # m/min
+dynamic_positioning_specs:
```

Comment:
Floating vessels can now have dynamic positioning specs defined. The class isn't used currently.

For OSS installation and turbine installation, a vessel either needs a jacking system or a dynamic positioning specs. I added new tests that verify that floating WTIV/feeders can install OSS and turbines